pgmpy/independencies/Independencies.py

```diff
@@ -49,6 +49,9 @@ def __str__(self):
         return string
 
     __repr__ = __str__
+    
+    def __eq__(self, other):
+        return self.__str__()==other.__str__()
```

Comment:
@ankurankan I tried equating the ``independencies`` attribute of the objects but I did not get the correct results. So, I had to compare the objects through their ``__str__`` methods.